config/config_test.go

```diff
@@ -301,6 +301,7 @@ func TestGatewayConfig_ToEtcdConfig(t *testing.T) {
 		},
 	}
 	for _, tt := range tests {
+		// nolint:scopelint
```

Comment:
i'd prefer just a tt := tt instead of nolint... but kind of a nit in a test file... i'm only saying this once, there are quite a few occurrences.

Reply:
cool, will change.  How is that syntax not considered a shadow declaration?  Is it just because the original `tt` isn't used anywhere in the loop before it's reassigned.